_docs/getting_started/usage.md

```diff
@@ -125,3 +130,12 @@ Example:
 ```
 $ lounge --version
 ```
+
+## Environment variables
+
+The Lounge is also configurable through the use of environment variables. CLI options will always have
```

Comment:
I think this should be reworded to say that home path is configurable via ENV param. Everything else should be configured via the config.